### ISSUE TITLE
Show days in previous month and next month 

### ISSUE THREAD

**Issue by nirpan** [nathanreyes/v-calendar#334]
I am using the latest beta version of v-calendar `(v1.0.0-beta.14)` with `Vue 3.0.0-rc.10`. In the stable version, some days in the previous and next month are visible in a slightly greyed out colour but on the beta version, the default behaviour seems to be to hide these days for previous and next month with `vc-opacity-0` which leaves a full row of empty white space as you can see [in this fiddle](https://jsfiddle.net/m9bsqk3r/1/). 

Is there an attribute or option to either completely remove the previous/next days or display these days in a greyed out colour as in the stable version?

**Comment by sigjak** [comment]
I agree that showing the previous/next days greyed out as used to be the case would be an improvement.

**Comment by benliu723** [comment]
It cost a lot of time to find out this issue, hope this improvement on the fly asap!

**Comment by ptdev** [comment]
It's hidden with CSS, so we can use CSS to make them visible again.
This wasn't very well tested, but it works on my end. The first instruction makes the days visible, the second one makes the days clickable.

```
.vc-day .vc-opacity-0 {
  opacity: 0.5;
}

.vc-day .vc-pointer-events-none {
  pointer-events: all;
}
```

**Comment by mjj999123** [comment]
In fact, you can choose display or hide previous and next months dates as you wish.
Just by overriding theme definition.
[customizable theme](https://github.com/nathanreyes/v-calendar/blob/master/src/utils/defaults/theme.js)
```
<template>
  <v-calendar
    class="calendar"
    :theme='themeStyles'
    ...
  />
</template>
<script>
import * as moment from 'moment'

export default {
  data () {
    return {
      themeStyles: {
        dayNotInMonth: 'not-in-month'
      },
    }
  }
  ...
}
</script>
<style scoped>
.calendar >>> .not-in-month {
  color: rgba(0, 0, 0, 0.5); // change this
}
</style>
```

**Comment by sebastiaanluca** [comment]
I think this is now hidden with JS? The hack isn't work anymore and neither is this (with the new default class):

```css
>>> .vc-container .is-not-in-month {
    color: red;
}
```

**Comment by Zagamant** [comment]
This working for me just to show hidden days (you can play with opacity)
```CSS
<style>
.is-not-in-month * {
  opacity: 1 !important;
  pointer-events: inherit !important;
}
</style>
```
